Projects/UOContent/Gumps/PricedResurrectGump.cs

```diff
@@ -0,0 +1,178 @@
+using Server.Engines.Virtues;
+using Server.Misc;
+using Server.Mobiles;
+using Server.Network;
+
+namespace Server.Gumps;
+
+public class PricedResurrectGump : StaticGump<PricedResurrectGump>
+{
+    private readonly Mobile m_Healer;
+    private readonly int m_Price;
+
+    public PricedResurrectGump(Mobile owner, Mobile healer, int price)
+        : base(150, 50)
+    {
+        m_Healer = healer;
+        m_Price = price;
+    }
+
+    protected override void BuildLayout(ref StaticGumpBuilder builder)
+    {
+        builder.SetNoClose();
+
+        builder.AddPage(0);
+
+        builder.AddImage(0, 0, 3600);
+
+        builder.AddImageTiled(0, 14, 15, 200, 3603);
+        builder.AddImageTiled(380, 14, 14, 200, 3605);
+
+        builder.AddImage(0, 201, 3606);
+
+        builder.AddImageTiled(15, 201, 370, 16, 3607);
+        builder.AddImageTiled(15, 0, 370, 16, 3601);
+
+        builder.AddImage(380, 0, 3602);
+
+        builder.AddImage(380, 201, 3608);
+
+        builder.AddImageTiled(15, 15, 365, 190, 2624);
+
+        builder.AddRadio(30, 140, 9727, 9730, true, 1);
+        builder.AddHtmlLocalized(65, 145, 300, 25, 1060015, 0x7FFF); // Grudgingly pay the money
+
+        builder.AddRadio(30, 175, 9727, 9730, false, 0);
+        builder.AddHtmlLocalized(65, 178, 300, 25, 1060016, 0x7FFF); // I'd rather stay dead, you scoundrel!!!
+
+        // Wishing to rejoin the living, are you?  I can restore your body... for a price of course...
+        builder.AddHtmlLocalized(30, 20, 360, 35, 1060017, 0x7FFF);
+
+        // Do you accept the fee, which will be withdrawn from your bank?
+        builder.AddHtmlLocalized(30, 105, 345, 40, 1060018, 0x5B2D);
+
+        builder.AddImage(65, 72, 5605);
+
+        builder.AddImageTiled(80, 90, 200, 1, 9107);
+        builder.AddImageTiled(95, 92, 200, 1, 9157);
+
+        builder.AddLabel(90, 70, 1645, m_Price.ToString());
+        builder.AddHtmlLocalized(140, 70, 100, 25, 1023823, 0x7FFF); // gold coins
+
+        builder.AddButton(290, 175, 247, 248, 2);
+
+        builder.AddImageTiled(15, 14, 365, 1, 9107);
+        builder.AddImageTiled(380, 14, 1, 190, 9105);
+        builder.AddImageTiled(15, 205, 365, 1, 9107);
+        builder.AddImageTiled(15, 14, 1, 190, 9105);
+        builder.AddImageTiled(0, 0, 395, 1, 9157);
+        builder.AddImageTiled(394, 0, 1, 217, 9155);
+        builder.AddImageTiled(0, 216, 395, 1, 9157);
+        builder.AddImageTiled(0, 0, 1, 217, 9155);
+    }
+
+    public override void OnResponse(NetState state, in RelayInfo info)
+    {
+        var from = state.Mobile;
+
+        from.CloseGump<PricedResurrectGump>();
```

Comment:
Where was this before? Because It is effectively closed at this point. Do we want to add it to the SendTo override instead?

Reply:
good call, it was in the top of the handler, I considered that as well, but it's been a while...